### ISSUE TITLE
Exclude all tests from the package

### ISSUE THREAD

**Issue by moubctez** [facelessuser/soupsieve#111]
Please, exclude all tests from the package. Otherwise they'll get installed in `site-packages/tests`.

```
--- setup.py.orig	2019-02-19 09:37:15.000000000 +0000
+++ setup.py
@@ -51,7 +51,7 @@ setup(
     author='Isaac Muse',
     author_email='Isaac.Muse@gmail.com',
     url='https://github.com/facelessuser/soupsieve',
-    packages=find_packages(exclude=['tests', 'tools']),
+    packages=find_packages(exclude=['tests', 'tests.*', 'tools']),
     install_requires=get_requirements(),
     license='MIT License',
     classifiers=[
```

**Comment by facelessuser** [comment]
Hmm, I didn't realize it worked that way...

Looks like it may be better just to explicitly tell it the packages:

```
packages=['soupsieve']
```

I'll look into this.  I didn't realize this is what we were doing.

**Comment by facelessuser** [comment]
This project is simple enough that explicitly defining the package is more than sufficient. We have a fix via PR #113 .

Thanks for pointing this out....now I have to update some of my other projects ☹️ .

**Comment by facelessuser** [comment]
It seems that that what `find_packages` does is find the files or modules. In our case, the sub modules are treated separately. `tests` only excludes the top level package and its files, but does not exclude sub modules.  A pattern of like so will actually exclude all tests and tools that we currently have:

```python
>>> import setuptools
>>> setuptools.find_packages(exclude=['test*', 'tools'])
['soupsieve']
>>>
```

It's a bit counter intuitive as normally I would expect if the top level module was excluded, all sub modules would also be excluded, but this is not the case.
